src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export { default as Code } from './Code';
```

Comment:
Delete this file. Since these components aren't really related, let's optimize for tree-shaking by making imports explicit and avoid foot-gunning users.

Reply:
@eliperelman In that case importing a component will be `import ErrorPanel from '@mozilla-frontend-infra/components/build/<component>'`. I don't like that we have to specifiy `build`. Are you okay if we rename the output directory to `core`?

```jsx
import ErrorPanel from '@mozilla-frontend-infra/components/core/<component>'

// ES5 versions of components
import ErrorPanel from '@mozilla-frontend-infra/components/es5/<component>'

```

Reply:
Any idea how material-ui does this?

Reply:
Ah, they output a directory in the root for every component it looks like.

Reply:
Interesting:

https://github.com/mui-org/material-ui/blob/master/packages/material-ui/scripts/copy-files.js#L19

They copy the package.json from the root into the build directory, along with README and friends, then publish the `build` directory:

https://github.com/mui-org/material-ui/blob/master/packages/material-ui/package.json#L30

Reply:
Material UI copies the `package.json`, `CHANGELOG`, `README.md`, `LICENSE`, `index.js`, `index.es.js` into `build/`. Next, all of the components are in their own directory i.e., `build/<component-directory>`. In addition to that they have a `build/es` which is a directory of all the components compiled for es5.

At the end of that, only `build/` is published to the npm registry.

Reply:
Material UI also appears to allow users to import components via `import { ComponentName } from '...'` https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/index.js.

Reply:
@eliperelman I think we should keep `src/components/index.js` for now and perhaps improve the workflow later. Thoughts?

Reply:
That's fine, but I would like to not release this until then. We can iterate, but let's hold off on publishing until we can clean that up.

Reply:
Sounds good to me.